test/squish/suite_VerifySettingsDialog/tst_VerifySettingsDialog/test.py

```diff
@@ -0,0 +1,25 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# -*- coding: utf-8 -*-
+# mypy: disable-error-code="attr-defined"
+
+import gui_helpers
+import test
+
+profileName = "deadlinecloud\\_squish"
+jobHistDir = "~/.deadline/job_history/deadlinecloud_squish"
+farmName = "Deadline Cloud Squish Farm"
+queueName = "Squish Automation Queue"
+jobAttachments = "COPIED"
+storageProfile = "Squish Storage Profile"
+conflictResOption = "NOT\\_SELECTED"
+loggingLevel = "WARNING"
+
+
+def main():
+    gui_helpers.launchDeadlineConfigGUI()
+    gui_helpers.setGlobalSettings(profileName, jobHistDir)
+    gui_helpers.setProfileSettings(farmName)
+    gui_helpers.setFarmSettings(queueName, storageProfile, jobAttachments)
+    gui_helpers.setGeneralSettings(conflictResOption, loggingLevel)
+    test.log("All settings config have been applied.")
+    gui_helpers.closeDeadlineConfigGUI()
```

Comment:
I'm a little confused about these functions. It seems like we're using them as tests but they also seem like setup for a test. Like none of them seem to validate that the controls have the values we expect after they're set. Or are these here just to setup before tests are run?

Reply:
These tests are run from a customer-facing perspective (so in essence this is a "setup" flow for a customer). The functions will fail if the test cannot find those values (thereby verifying if the Dialogue is able to pull the correct farm info, etc).